content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -192,6 +226,12 @@ Or:
 $ kubectl delete ns istio-operator --grace-period=0 --force
 {{< /text >}}
 
+If you just want to uninstall old Istio control plane but keep the new one during canary upgrade, do not delete the in-cluster CR, otherwise the new control plane resources would be uninstalled as well. Run this command instead:
```

Comment:
Grammar.  "To uninstall a single control control but keep the new one, do not delete the in-cluster IstioOperator.  Run:"

Reply:
done